composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "openemr/openemr",
-    "version": "5.0.3",
+    "version": "6.0.0",
```

Comment:
It should be 6.0.0-dev or 6.0.0-RC.  IMO
This version declaration is about compatibility and constraint in module development.

Reply:
hi @zerai , are modules currently looking for this or would this be an enhancement?

Reply:
Hi! not an enhancement, [here](https://github.com/openemr/oe-module-faxsms/blob/ab000dd03ded7db28219c3a40def9330c20d19ff/composer.json#L34) the usage.
But you can ignore my comment module developers can use their own composer.json file to support or not the new version 6.